l1-contracts/deploy-scripts/DeployCTMUtils.s.sol

```diff
@@ -90,9 +92,6 @@ struct ContractsConfig {
     uint256 genesisRollupLeafIndex;
```

Comment:
I think ideally we would have one file with all the address/config structs, as it avoids duplication, and it creates a clear mental structure